src/Traits/Modifiable.php

```diff
@@ -15,8 +15,6 @@ trait Modifiable
 		ModifiesLabels::__construct as private __mlConstruct;
 	}
 
-	private $messageRequest;
```

Comment:
This is the one you are saying that's throwing exceptions?

Reply:
This is one of three that is causing issues. This property isn't used in this trait. It is defined in `Traits\ModiefiesLabels` and is used in that trait.